xml/System.Runtime.InteropServices/CharSet.xml

```diff
@@ -94,7 +94,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>Marshal strings as multiple-byte character strings.</summary>
+        <summary>Marshal strings as multiple-byte character strings: ANSI on Windows, and UTF-8 on Unix.</summary>
```

Comment:
Broadly speaking, this is true but there is a little more nuance. Technically it is the system default ANSI code page - see the description of `CP_ACP` in https://docs.microsoft.com/windows/desktop/api/stringapiset/nf-stringapiset-widechartomultibyte.

Reply:
If interested, the specific code that does this marshaling is at:

Windows: https://github.com/dotnet/coreclr/blob/07b3afc27304800f00975c8fd4836b319aaa8820/src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.Windows.cs#L58-L87

Non-Windows:
https://github.com/dotnet/coreclr/blob/07b3afc27304800f00975c8fd4836b319aaa8820/src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.Unix.cs#L39-L53

Reply:
The best solution may be to add more remarks in [Charsets and marshaling](https://github.com/dotnet/docs/blob/staging/docs/standard/native-interop/charset.md) and add a reference to it here (like #2342).

Reply:
@NextTurn I think that would be a reasonable solution.